exampleAudioSink/src/main/java/com/twilio/exampleaudiosink/MediaPlayerHelper.java

```diff
@@ -0,0 +1,51 @@
+package com.twilio.exampleaudiosink;
+
+import android.media.AudioManager;
+import android.media.MediaPlayer;
+
+import java.io.IOException;
+
+class MediaPlayerHelper {
+
+    private MediaPlayer player;
+    private boolean isReleased = false;
+    private MediaPlayerListener listener;
+    MediaPlayerHelper() {
+    }
+
+    boolean isPlaying() {
+        if (player == null || isReleased) {
+            return false;
+        }
+        return player.isPlaying();
+    }
+
+    void playFile(String path, final MediaPlayerListener listener) throws IOException {
```

Comment:
I think we can get rid of the `MediaPlayerListener` interface by passing the OnCompletionListener in directly.